modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/java/com/liferay/frontend/editor/ckeditor/sample/web/internal/display/context/CKEditorSampleDisplayContext.java

```diff
@@ -5,16 +5,59 @@
 
 package com.liferay.frontend.editor.ckeditor.sample.web.internal.display.context;
 
+import com.liferay.client.extension.constants.ClientExtensionEntryConstants;
+import com.liferay.client.extension.type.EditorConfigContributorCET;
+import com.liferay.client.extension.type.manager.CETManager;
 import com.liferay.frontend.taglib.clay.servlet.taglib.util.TabsItem;
 import com.liferay.frontend.taglib.clay.servlet.taglib.util.TabsItemListBuilder;
+import com.liferay.portal.kernel.dao.orm.QueryUtil;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.portal.vulcan.pagination.Pagination;
 
 import java.util.List;
 
+import javax.portlet.RenderRequest;
+
 /**
  * @author Marko Cikos
  */
 public class CKEditorSampleDisplayContext {
 
+	public CKEditorSampleDisplayContext(
+		CETManager cetManager, RenderRequest renderRequest) {
+
+		_cetManager = cetManager;
+		_renderRequest = renderRequest;
+
+		_themeDisplay = (ThemeDisplay)renderRequest.getAttribute(
+			WebKeys.THEME_DISPLAY);
+	}
+
+	public JSONArray getEditorTransformerURLsJSONArray() throws Exception {
+		return JSONUtil.toJSONArray(
+			_cetManager.getCETs(
+				_themeDisplay.getCompanyId(), null,
+				ClientExtensionEntryConstants.TYPE_EDITOR_CONFIG_CONTRIBUTOR,
+				Pagination.of(QueryUtil.ALL_POS, QueryUtil.ALL_POS), null),
+			cet -> {
+				EditorConfigContributorCET editorConfigContributorCET =
+					(EditorConfigContributorCET)cet;
+
+				if (StringUtil.matches(
+						editorConfigContributorCET.getEditorConfigKeys(),
+						"sampleReactClassicEditor")) {
```

Comment:
For react usages there is no way for config contribs to kick in :( 

We knew this when implemented the CX, but this tells me we might pay attention to how we can easily get the right CXs affecting a given editor occurrence implemented in react (future task though)